whotracksme/website/build/trackers.py

```diff
@@ -152,6 +154,22 @@ def tracker_header_stats(apps):
         "data": mean(data)
     }
 
+
+def og_snippet_tracker_page(data, aid, app):
+    prevalence_stats = prevalence(app)
+
+    og = OG_SNIPPETS.get('tracker-page').copy()
+    og['url'] = og['url'] + data.url_for('app', aid).replace('./', '')
+    og['title'] = og['title'] + app['name']
+    og['description'] = "{0} is detected tracking {1}% of web traffic and is present on {2}% of known sites. Tracker category: {3}".format(
+        app['name'],
+        prevalence_stats['pages'],
+        prevalence_stats['domains'],
+        app['cat'].replace("_", " ").capitalize() + " - " + CATEGORY_DESC[app['cat']]
+    )
```

Comment:
Since you need to specialize the `og.json` template, you could have a function to handle the boilerplate:
```python
def specialize_og_data(entity, specialize):
    return {
        key: specialize.get(key, lambda v: v)(value)
        for key, value in OG_SNIPPETS[entity].items()
    }
```

Which you could use like:
```python
def og_snippet_tracker_page(data, aid, app):
    prevalence_stats = prevalence(app)
    return specialize_og_data('tracker-page', {
        'url': lambda t: t.format(url=data.url_for('app', aid).replace('./', '')),
        'title': lambda t: t.format(name=app['name']),
        'description': lambda t: t.format(
            name=app['name'],
            pages=prevalence_stats['pages'],
            domains=prevalence_stats['domains'],
            category=app['cat'].replace("_", " ").capitalize() + " - " + CATEGORY_DESC[app['cat']]
    }))
```